libmscore/arpeggio.h

```diff
@@ -48,13 +48,16 @@ class Arpeggio final : public Element {
       virtual QPointF gripAnchor(Grip) const override;
       virtual void startEdit(EditData&) override;
 
+      static const std::array<const char*, 6> arpeggioTypeNames;
+
    public:
       Arpeggio(Score* s);
       virtual Arpeggio* clone() const override    { return new Arpeggio(*this); }
       virtual ElementType type() const override   { return ElementType::ARPEGGIO; }
 
       ArpeggioType arpeggioType() const    { return _arpeggioType; }
       void setArpeggioType(ArpeggioType v) { _arpeggioType = v;    }
+      const char* arpeggioTypeName()       { return arpeggioTypeNames[int(_arpeggioType)]; }
```

Comment:
and return `QApp->translate("Palette", arpeggioTypeNames[int(_arpeggioType)};` here